README.md

```diff
@@ -9,167 +10,191 @@ npm install --save bs-faker
+let dataUri = BsFaker.Image.dataUri(~width: int=?, ~height: int=?, ()); /* => string; "data:inage/svg+xml;..." */
+let fashion = BsFaker.Image.fashion(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let food = BsFaker.Image.food(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let image = BsFaker.Image.image(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let imageUrl = BsFaker.Image.imageUrl(~width: int=?, ~height: int=?, ~category: string=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/640/480" */
+let nature = BsFaker.Image.nature(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let nightlife = BsFaker.Image.nightlife(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let people = BsFaker.Image.people(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let sports = BsFaker.Image.sports(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let technics = BsFaker.Image.technics(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+let transport = BsFaker.Image.transport(~width: int=?, ~height: int=?, ~randomize: bool=?, ()); /* => string; "http://lorempixel.com/..." */
+```
+
+### `BsFaker.Name`
 
 ```reason
-let findName = Faker.Name.findName(~firstName: string=?, ~lastName: string=?, ~gender: string=?, ()); /* => string; "Mollie Hills" */
-let firstName = Faker.Name.firstName(~gender: string=?, ()); /* => string; "Max" */
-let lastName = Faker.Name.lastName(~gender: string=?, ()); /* => string; "Bartoletti" */
-let prefix = Faker.Name.prefix(~gender: string=?, ()); /* => string; "Dr." */
-let suffix = Faker.Name.suffix(); /* => string; "MD" */
-let title = Faker.Name.title(); /* => string; "Senior Creative Specialist" */
-let jobArea = Faker.Name.jobArea(); /* => string; "Tactics" */
-let jobType = Faker.Name.jobType(); /* => string; "Assistant" */
+let findName = BsFaker.Name.findName(~firstName: string=?, ~lastName: string=?, ~gender: string=?, ()); /* => string; "Mollie Hills" */
+let firstName = BsFaker.Name.firstName(~gender: string=?, ()); /* => string; "Max" */
+let lastName = BsFaker.Name.lastName(~gender: string=?, ()); /* => string; "Bartoletti" */
+let prefix = BsFaker.Name.prefix(~gender: string=?, ()); /* => string; "Dr." */
+let suffix = BsFaker.Name.suffix(); /* => string; "MD" */
+let title = BsFaker.Name.title(); /* => string; "Senior Creative Specialist" */
+let jobArea = BsFaker.Name.jobArea(); /* => string; "Tactics" */
+let jobType = BsFaker.Name.jobType(); /* => string; "Assistant" */
 ```
 
-### `Faker.Internet`
+### `BsFaker.Internet`
```

Comment:
Looks like this section should be removed (maybe due to conflict?)

Reply:
Oops, conflict merging error...